### PR TITLE
Tyler dev

### PR DESCRIPTION
**Pull request by naruto11236**
The submit buttons should be working and fixed css. Log in page not done but created.

### CODE REVIEW

File: src/main/java/NewMenteeServlet.java [hunk 1]
```diff
@@ -42,6 +43,6 @@ public void doPost(HttpServletRequest request, HttpServletResponse response) thr
             .build();
     datastore.put(menteeEntity);
 
-    response.sendRedirect("/index.html");
+    //response.sendRedirect("/matchingPage.html");
```
Comment (emilyzhang242) [inline src/main/java/NewMenteeServlet.java]: It looks like this code was moved to the area above, so please delete. We want to try to keep the code clean and not to merge code to the master branch unless it's ready, so other people won't be confused by random commented out code.

File: src/main/webapp/default.css [hunk 2]
```diff
@@ -228,10 +228,11 @@ hr {
 	overflow: hidden;
 	/*background: url(images/bg01.png) repeat;*/
 	background-size: 70%;
-	padding: 6em 0em 2em 0em;
+    /*padding: 6em 0em 2em 0em;*/
```
Comment (emilyzhang242) [inline src/main/webapp/default.css]: Why is some of the CSS commented out here?

File: src/main/webapp/index.html [hunk 3]
```diff
@@ -49,8 +49,8 @@ <h2>MentorMatch</h2>
 </div>
 	
 <!--
-<div id="footer-wrapper">
-	<div id="footer" class="container">
+<div id="footer-wrapper"
```
Comment (emilyzhang242) [inline src/main/webapp/index.html]: You need to have closing brackets, so the original code should be correct. Please revert.

File: src/main/webapp/mentor.html [hunk 4]
```diff
@@ -45,7 +67,8 @@ <h1 class = "title">Mentor Profile</h1>
                 <br/>
                 <br/>
                 <input id="submitButton" class="input" type="submit" />
-           </form>
+
```
Comment (emilyzhang242) [inline src/main/webapp/mentor.html]: Super nitpick: remove extra space

File: src/main/webapp/mentor.html [hunk 5]
```diff
@@ -6,7 +6,28 @@
     <link rel="stylesheet" href="mentor_signup_style.css">
     <link rel="preconnect" href="https://fonts.gstatic.com">
     <link href="https://fonts.googleapis.com/css2?family=Raleway&display=swap" rel="stylesheet">
-    
+   <script>
+       function saveValues() {
+           //window.location.hrep = "matchingPage.html";
```
Comment (emilyzhang242) [inline src/main/webapp/mentor.html]: Why is this code commented out? Are you waiting on something to have it work?

**Review by emilyzhang242 — CHANGES_REQUESTED** [review]
Overall looks good! Just some comments and questions regarding commented out code mostly.

File: target/classes/META-INF/resources/matchingPage.html [hunk 6]
```diff
@@ -0,0 +1,141 @@
+<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Strict//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-strict.dtd">
```
Comment (emilyzhang242) [inline target/classes/META-INF/resources/matchingPage.html:1]: Is this matchingPage file also in the webapp folder? Have you tried deleting the extra repeated files in META-INF and seeing if the code still works? 

I wonder if something strange happened with the merge to cause this.

File: target/classes/META-INF/resources/default.css [hunk 7]
```diff
@@ -582,7 +582,51 @@ hr {
 		padding-left: 40px;
 	}
 
+/*********************************************************************************/
```
Comment (emilyzhang242) [inline target/classes/META-INF/resources/default.css:585]: Can you include screenshots of any UI where large changes have been made (except for something trivial like the spelling mistake you corrected)? This makes it easy for others who are reviewing your code to spot issues with the design/UI which is really hard to do just by looking at the code alone.

**Review by emilyzhang242 — CHANGES_REQUESTED** [review]
Overall looks good! Main thing is trying to delete the META-INF files and seeing if that breaks anything. I feel like it shouldn't, but if it does, then we can try digging a little deeper. Before deleting, I would also do a sweep of the files just to make sure none are missing from the webapp folder and that the merged code has been copied into the folder correctly as well and not just META-INF. 

Let me know if you run into issues.

File: target/classes/META-INF/resources/matchingPage.html (hunk 6 above)
Comment (MariferSalas) [reply target/classes/META-INF/resources/matchingPage.html:1]: Yes, the matchingPage file is also in the webapp folder. I tried to delete them but everytime I run the command "mvn package exec:java" appeared again.

I'll try to uncommit those changes and ignore the files.

**Pull request merged**